src/cloud_run/utils/helpers.py

```diff
@@ -0,0 +1,22 @@
+import hashlib
+from typing import NamedTuple
+
+
+class TollwayEvent(NamedTuple):
+    year: str
+    make: str
+    model: str
+    category: str
+    license_plate: str
+    vin: str
+    state: str
+    primary_color: str
+    tollway_state: str
+    tollway_name: str
+    timestamp: str
```

Comment:
datetime?

Reply:
possibly. I think other program that generates the messages has str type for the timestamp key. I'll doublecheck to make sure using correct type.